### PR TITLE
OADP -3250 Updated1 - OADP ROSA docs xref mob docs

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP - 1.3.1

OCP - 4.12+

Resolves - https://issues.redhat.com/browse/OADP-3250

Deploy preview - https://69714--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications

### CODE REVIEW

File: modules/performing-a-backup-oadp-rosa-sts.adoc
Comment (CarmiWisemon) [inline modules/performing-a-backup-oadp-rosa-sts.adoc:152]: I can't link in the documentation to Git.
Are there published documents that I can use instead or should I remove these links?

Comment (kaovilai) [reply modules/performing-a-backup-oadp-rosa-sts.adoc:155]: https://red.ht/oadp-faq

Does this work?

File: backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc
Comment (kaovilai) [inline backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc:17]:
```suggestion
After you create your clusters, you can operate your clusters with the OpenShift web console or through link:https://docs.openshift.com/dedicated/ocm/ocm-overview.html[{cluster-manager-first}]. You can also use ROSA service via OpenShift APIs and command-line interface (CLI) tools.
```

File: modules/performing-a-backup-oadp-rosa-sts.adoc
Comment (CarmiWisemon) [reply modules/performing-a-backup-oadp-rosa-sts.adoc:155]: Yes, thank you!

File: backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc
Comment (CarmiWisemon) [reply backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc]: @kaovilai 
I made the changes and suggestions you requested.
Do you have any other comments?

File: modules/installing-oadp-rosa-sts.adoc
Comment (PrasadJoshi12) [reply modules/installing-oadp-rosa-sts.adoc:69]: @CarmiWisemon please remove the *

File: modules/installing-oadp-rosa-sts.adoc
Comment (PrasadJoshi12) [inline modules/installing-oadp-rosa-sts.adoc:140]: @CarmiWisemon this should be point no. 5, right ? Currently it's rendered as 1

File: modules/installing-oadp-rosa-sts.adoc
Comment (PrasadJoshi12) [inline modules/installing-oadp-rosa-sts.adoc:182]: it should be point 1.b, right ? currently both points are rendered as a in preview.

File: modules/cleanup-a-backup-oadp-rosa-sts.adoc
Comment (PrasadJoshi12) [inline modules/cleanup-a-backup-oadp-rosa-sts.adoc:7]: Title and description is misleading. This section contains information on how can we cleanup backup as well as the OADP operator and related CR's from the cluster.

Comment (CarmiWisemon) [reply modules/cleanup-a-backup-oadp-rosa-sts.adoc:7]: @PrasadJoshi12 

Should it be as follows:

Title - Cleaning a cluster after a backup with OADP and ROSA STS

Description - After you complete a backup with {oadp-first} and {product-rosa} (ROSA) {sts-full} (STS), you need to remove from the cluster the data of the backup, OADP Operator and other related custom resources (CRs).

Comment (PrasadJoshi12) [reply modules/cleanup-a-backup-oadp-rosa-sts.adoc:7]: Imo Description should be something like this. 
If you want to uninstall the OADP operator along with the backups and S3 bucket, follow below instructions.

Comment (CarmiWisemon) [reply modules/cleanup-a-backup-oadp-rosa-sts.adoc:7]: @PrasadJoshi12 
Thank you!

File: backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc
Comment (kaovilai) [inline backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc:33]:
```suggestion
== Example: Backing up workload on OADP ROSA STS, with optional cleanup.
```

I feel this might have clearer messaging which tell cu that cleaning up is optional. Perhaps when this merge, we can sync the changes here back to mobb docs.

File: modules/cleanup-a-backup-oadp-rosa-sts.adoc
Comment (kaovilai) [inline modules/cleanup-a-backup-oadp-rosa-sts.adoc:9]:
```suggestion
If you need to uninstall the OpenShift API for Data Protection (OADP) Operator together with the backups and the S3 bucket from this example, follow these instructions.
```

File: backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc
Comment (kaovilai) [inline backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc:13]: Out of curiosity, where are these statements coming from?

Comment (CarmiWisemon) [reply backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc:13]: I received them from someone else.
I think they are coming from here.
https://docs.openshift.com/rosa/rosa_architecture/rosa-understanding.html

Comment (kaovilai) [reply backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc:13]: Ok cool. Dug deeper and found
https://aws.amazon.com/rosa/ and https://aws.amazon.com/products/frontend-web-mobile/

I thought they meant ocp has something mobile related.

Thanks!

File: modules/preparing-aws-credentials-for-oadp.adoc
Comment (kaovilai) [inline modules/preparing-aws-credentials-for-oadp.adoc:150]: I think this needs to be similar to [L97](https://github.com/openshift/openshift-docs/pull/69714/files#diff-077ce22b47a7ee6633e514417f70b84968b4b8bbca5371b0672079d4fd748798R97) where it has to be in the same line

```suggestion
  --tags Key=rosa_cluster_id,Value=${ROSA_CLUSTER_ID} Key=rosa_openshift_version,Value=${CLUSTER_VERSION} Key=rosa_role_prefix,Value=ManagedOpenShift Key=operator_namespace,Value=openshift-adp Key=operator_name,Value=openshift-oadp \
```

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: modules/preparing-aws-credentials-for-oadp.adoc
Comment (CarmiWisemon) [reply modules/preparing-aws-credentials-for-oadp.adoc:150]: @kaovilai 
Just the --tags line or also the --assume line above it and the --query line at the bottom?

Comment (kaovilai) [reply modules/preparing-aws-credentials-for-oadp.adoc]: looking

Comment (CarmiWisemon) [reply modules/preparing-aws-credentials-for-oadp.adoc]: Key=rosa_cluster_id,Value=${ROSA_CLUSTER_ID}
Key=rosa_openshift_version,Value=${CLUSTER_VERSION}
Key=rosa_role_prefix,Value=ManagedOpenShift
Key=operator_namespace,Value=openshift-adp
Key=operator_name,Value=openshift-oadp \

**Review by kaovilai — APPROVED** [review]

**Review by skopacz1 — COMMENTED** [review]
Left some comments, but nicely done with such a large chunk of content! Let me know if you have any questions about my feedback.

Also, I'm still relatively new to peer reviewing, so please wait until @adellape approves my review before implementing any feedback. Thanks!

File: modules/installing-oadp-rosa-sts.adoc
Comment (adellape) [reply modules/installing-oadp-rosa-sts.adoc:64]: @CarmiWisemon Is this because you're wanting to hard-link to OCP docs specifically from the ROSA docs?

**Review by adellape — COMMENTED** [review]
@skopacz1 Review lgtm!

File: modules/installing-oadp-rosa-sts.adoc
Comment (CarmiWisemon) [reply modules/installing-oadp-rosa-sts.adoc:14]: We don't usually use the short attributed for AWS and STS because it's shorter to write AWS and STS, but I'll change it.

File: modules/installing-oadp-rosa-sts.adoc
Comment (CarmiWisemon) [reply modules/installing-oadp-rosa-sts.adoc:137]: Thank you. I placed them in a note.

File: modules/installing-oadp-rosa-sts.adoc
Comment (CarmiWisemon) [reply modules/installing-oadp-rosa-sts.adoc:142]: Thank you. I used the 2nd option.

File: modules/performing-a-backup-oadp-rosa-sts.adoc
Comment (CarmiWisemon) [reply modules/performing-a-backup-oadp-rosa-sts.adoc:158]: Good idea.

File: modules/installing-oadp-rosa-sts.adoc
Comment (PrasadJoshi12) [inline modules/installing-oadp-rosa-sts.adoc:69]: @CarmiWisemon If we are installing the Operator via UI(Operatorhub) on 4.14+ cluster then it requires role_ARN field. Sorry I just noticed it. 

sts

File: modules/installing-oadp-rosa-sts.adoc
Comment (CarmiWisemon) [reply modules/installing-oadp-rosa-sts.adoc:64]: @adellape It's because we want to hard-link from the ROSA docs to OCP.

**Comment by CarmiWisemon** [comment]
> Left some comments, but nicely done with such a large chunk of content! Let me know if you have any questions about my feedback.
> 
> Also, I'm still relatively new to peer reviewing, so please wait until @adellape approves my review before implementing any feedback. Thanks!

@skopacz1 @adellape 

Thank you for your feedback. I implemented your suggestions and another request that came from QE.
Thank you for your valuable suggestions and feedback.
Carmi

**Pull request merged**
